### DISCUSSION TITLE
Make `@vitejs/plugin-react` optional peer dependency

### DISCUSSION THREAD

**Discussion by ahnpnl** [storybookjs/builder-vite#555] [Ideas]
Hi, 

Is it possible to make `@vitejs/plugin-react` as an optional peer dependency? I have Storybook using Vue and I don't have `@vitejs/plugin-react` in my lock file and my Storybook still works.

**Comment by IanVS** [comment]
Doh, yes it should be optional, sorry about that.  Would you like to submit a PR, @ahnpnl?

**Reply by IanVS** [reply]
Nevermind, I've added it as part of https://github.com/storybookjs/builder-vite/pull/552.  Thanks again for mentioning it.
